### PR TITLE
Fixes a few issues with playing cards

### LINKED ISSUE #7338: You can make infinite objects with the Wizard Card Deck
Steps to reproduce, sorry if its a little confusing:

Spaw /obj/item/toy/cards/wizard/full
Drag two cards
Make a "hand of cards" by merging a card with another card
Remove a card form the "hand"
Remove a card form the "hand"
You now have a red knuckle on where your "hand of cards" are supposed to be.

Shuffle the cards in the deck again and repeat to get another "hand".
You can delete the duplicate items by putting them inside the wizard deck

### LINKED ISSUE #10855: Playing cards not visible when stacked into a playing hand
(WEB REPORT BY: orzorn REMOTE: 172.93.109.202:7777)

> Revision (Should be above if you're viewing this from ingame!)

3b41948840cd36f803bf3d6ead4415df8b1189cd 

> General description of the issue
> A "hand" of cards, that is, multiple cards in a stack that you're supposed to be able to flip over and see while in your hand, do not show up as cards, and instead a strange red icon with faded out cards barely visible.
> 
> What you expected to happen
> To be able to see the hand of cards.
> 
> What actually happened
> They were nearly invisible and a strange red icon was present.
> 
> Steps to reproduce if possible
> Get a deck, draw more than one card and put them on each other to make a hand. Witness the strange icon and inability to see the cards.

### PR DESCRIPTION
**Pull request by JustSumBody**
fixes #10855 and an unreported issue that would let you bypass the hand's max through using single cards on hands. Also fixes some really funky shit where making a hand places the hand in both your hand slots, so drawing from the deck becomes impossible without dropping the whole thing or placing it back in the deck.
Maybe fixes #7338 ? I can't reproduce it at any rate.

~~My understanding of planes and layers is limited though, so this has the unintended result of making the cards layer above mobs when they're outside of your hand. I was told this usually resets upon being dropped, but I'm not seeing that happen.~~
image 

Ayy it's fixed now

### CODE REVIEW

File: code/modules/games/cards/playing_cards.dm [hunk 1]
```diff
@@ -276,6 +276,7 @@
 		var/obj/item/toy/singlecard/card = currenthand[i]
 		if(card)
 			card.layer = FLOAT_LAYER
+			card.plane = HUD_PLANE
```
Comment (ghost) [inline code/modules/games/cards/playing_cards.dm]: change this to FLOAT_PLANE ?

Comment (ghost) [inline code/modules/games/cards/playing_cards.dm]: objects may not be able to float plane. In that case; turn the obj into an image with the same appearance and use that instead.

Comment (PJB3005) [inline code/modules/games/cards/playing_cards.dm]: I don't see why they wouldn't be able to have `FLOAT_PLANE`.

Comment (ghost) [inline code/modules/games/cards/playing_cards.dm]: @PJB3005 eh I've had a problem like that before at some point I think

**Comment by ghost** [comment]
@JustSumBody did that fix it?

**Comment by JustSumBody** [comment]
Yes

**Comment by clusterfack** [comment]
I can't get a feel just from reading this but I'm sure you tested it

**Pull request merged**